explainaboard/processors/text_pair_classification.py

```diff
@@ -115,14 +115,14 @@ def default_metrics(
 
     @aggregating()
     def _statistics_func(self, samples, sys_info: SysOutputInfo):
-        return {
-            'source_vocab': accumulate_vocab_from_samples(
-                samples, lambda x: x['text1'], unwrap(sys_info.source_tokenizer)
-            ),
-            'target_vocab': accumulate_vocab_from_samples(
-                samples, lambda x: x['text2'], unwrap(sys_info.target_tokenizer)
-            ),
-        }
+
+        source_vocab, source_vocab_rank = accumulate_vocab_from_samples(
```

Comment:
I understand why you combined these together, but I think it'd be better to calculate the vocabularies separately. For example, there might be a cross-lingual text pair classification task where the source and target vocabs would be in different languages.